### DISCUSSION TITLE
I am verified as student but not getting any benefits please help !

### DISCUSSION THREAD

**Discussion by dharamLearn** [community/community#56101] [GitHub Education] [answered]
### Hi everyone,

Hi, I am verified as student but not getting any benifit of student developer pack, I am getting this message from the last 3 days

"Hi, Dharamveer Singh! You were last verified as a student on May 20, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."

Please help me.

**Comment by MaxJ02** [comment, marked as answer]
I'm in the same boat and it seems that you have to let at least 72 hours pass before the benefits get activated. It also seems like you will get an email when your benefits finally activate, so just be patient.

**Comment by katikuuu** [comment]
Try logging in to your Visual Studio code using your GitHub account then download GitHub Copilot Extension and check its availability.It will work without the Wait time

**Reply by EsqerYasen** [reply]
it's not in my case .  i still can't use coilot . i was approved 3day's ago.

**Reply by EsqerYasen** [reply]
Extension activation failed: "No access to GitHub Copilot found."

**Reply by EsqerYasen** [reply]
i just tried logout and login to VScode with github account and working just fine !

**Reply by vedantmishra03** [reply]
Hello, it's not working for me. I added my college domain ID to my personal GitHub account and then claimed to get the student developer pack. However, for the last two days, they have been showing that I am approved but do not have access to the student developer pack. Additionally, I am unable to get the details of compiling in my VS Code , what should I do now ???

**Reply by katikuuu** [reply]
Did you get the Student Developer pack
@EsqerYasen @vedantmishra03

**Reply by wijrl** [reply]
Yes cause he has a "pro" badge on his profile.

**Reply by mrlufus** [reply]
The Login/Logout Trick no longer works. Tested it today. Have the Student Pack already for 2 Days.

**Comment by parthmern** [comment]
i also got approved on 9th feb but still i am not getting benefits

**Reply by vedantmishra03** [reply]
Hello , just wait for 2-3 working days definitely you will get it.

**Reply by parthmern** [reply]
It's already verified on 9th feb and today is 12th feb... But still waiting hope it will be verified soon... Thanks

**Reply by parthmern** [reply]
It's verified now.. I just receive the mail

**Comment by alsheroziy** [comment]
Hello everyone, I just joined the github community, please tell me the rules

**Comment by nandasafiqalfiansyah** [comment]
same problem i can wait for 4 day

**Comment by modeitsch** [comment]
I have the same problem I'm waiting for almost a week

**Comment by tigazyyy** [comment]
i also got approved on Dec 29, 2024 but still i am not getting benefits

**Reply by samus-aran** [reply]
It states that you have been approved. What benefits are you seeking to redeem?

**Reply by tigazyyy** [reply]
when i discuss in this topic, the benefit is approved to me in a soon! Thanks a lot !
